### PR TITLE
add sqs publishing and consuming

### LINKED ISSUE #7: add SQS consumer
Add writing to SQS (probably same as we write to Kafka) and a SQS consumer.
Pass the propagated header as a message attribute.

### PR DESCRIPTION
**Pull request by aviramha**
closes #7

### CODE REVIEW

File: ip-visit-counter/main.go [hunk 1]
```diff
@@ -148,10 +206,11 @@ func getCount(c *gin.Context) {
 		return
 	}
 
-	c.JSON(200, gin.H{"count": count, "text": ResponseString, "info": ipInfo})
+	c.JSON(200, gin.H{"count": count, "text": ResponseString + "hi from", "info": ipInfo})
```
Comment (t4lz) [inline ip-visit-counter/main.go]: Is this change intentional, or is this part of a demo that got committed accidentally?

File: ip-visit-sqs-consumer/ip-visit-sqs-consumer
Comment (t4lz) [inline ip-visit-sqs-consumer/ip-visit-sqs-consumer:1]: Committed by mistake, right? Should be added to .gitignore

File: ip-visit-sqs-consumer/mirrord.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "target": "deployment/ip-visit-sqs-consumer",
+    "feature": {
+        "split_queues": {
+            "ip-count": {
+                "queue_type": "SQS",
+                "message_filter": {
+                    "x-pg-tenant": "Avi.+"
```
Comment (t4lz) [inline ip-visit-sqs-consumer/mirrord.json]: Nitpick: the regex is matched anywhere within the value, so the `+` is not necessary, and including it could make people think the complete attribute value must match the pattern for the message to be matched.

```suggestion
                    "x-pg-tenant": "Avi."
```

File: ip-visit-counter/main.go (hunk 1 above)
Comment (aviramha) [reply ip-visit-counter/main.go]: reverted, ty

File: ip-visit-sqs-consumer/ip-visit-sqs-consumer
Comment (aviramha) [reply ip-visit-sqs-consumer/ip-visit-sqs-consumer:1]: done

**Review by t4lz — APPROVED** [review]
